cypress/support/Models.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+ */
+
+const helpers = require('./helpers.js')
+const homePage = require('./components/HomePage')
+const modelPage = require('./components/ModelPage')
+const entities = require('./Entities')
+const actions = require('./Actions')
+
+export function CreateNewModel(name)
+{
+  homePage.Visit()
+  homePage.ClickNewModelButton()
+  homePage.TypeModelName(name)
+  homePage.ClickSubmitButton()
+  modelPage.VerifyModelName(name)
+}
+
+export function ImportModel(modelNamePrefix, fileName)
+{
+  // Maximum Name Length is 30 Characters
+  const name = `${modelNamePrefix}-${helpers.ModelNameTime()}`
+
+  homePage.Visit()
+  homePage.ClickImportModelButton()
+
+  homePage.TypeModelName(name)
+  homePage.UploadImportModelFile(fileName)
+  homePage.ClickSubmitButton()
+
+  return name
+}
+
+
+// Old code that is likely to be removed or remodeled in the future ------------------------
```

Comment:
If not used and you don't plan to do so in the near future, I'd say delete it.  We can always grab from source control if we need it back

Reply:
Done